resources/memorytest/run.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+RESPONSE_FILE=response.txt
+
+#
+# The user must set these to valid values
+#
+LICENSE_FILE_PATH=
```

Comment:
you can ask the user for these if not set. Check https://github.com/curityio/nginx_phantom_token_module/blob/master/configure#L14 for an example